src/modules/lighthouse/TableGenerator.ts

```diff
@@ -0,0 +1,67 @@
+import { ShortReportObject, LighthouseReportDoc } from '../../clients/Lighthouse'
+
+export interface TableRow {
+  [title: string]: any
+}
+
+export class TableGenerator {
+  private cols = ['Performance', 'Accessibility', 'Best Practices', 'SEO']
+  private addedCols: string[] = []
+  private rows: TableRow[] = []
+
+  /**
+   * Insert the scores of a report to the table as a row
+   *
+   * @param report ShortReportObject array representing the scores of this report
+   */
+  public addReportScores(report: ShortReportObject[]) {
+    const row = {}
+    report.forEach(audit => {
+      row[audit.title] = audit.score
+    })
```

Comment:
Just out of curiosity:
```
const row = report.reduce((acc, audit) => {
  acc[audit.title] = audit.score
  return acc
}, {})
```

Reply:
Much better, I am not very used to this functional approach in javascript..